client/related_items/ItemCard.jsx

```diff
@@ -19,32 +20,37 @@ class ItemCard extends React.Component {
       .then(productInfo => this.setState({
         category: productInfo.category,
         name: productInfo.name,
-        rating: '4.5'
-      }))
-      .then(() => helpers.getProductStyles(this.props.id)
-        .then(productStyles => helpers.findDefaultStyle(productStyles.results)
-          .then(defaultStyle => {
-            let price, thumbnailUrl;
-
-            if (defaultStyle.sale_price === null) { price = defaultStyle.original_price; }
-            else { price = defaultStyle.sale_price; }
-
-            if (defaultStyle.photos[0].thumbnail_url === null) {
-              thumbnailUrl = 'https://cdn.shopify.com/s/files/1/0533/2089/files/placeholder-images-image_large.png?format=webp&v=1530129081';
-            } else { thumbnailUrl = defaultStyle.photos[0].thumbnail_url; }
-
-            this.setState({
-              thumbnailUrl: thumbnailUrl,
-              price: price
-            });
-          })
-        )
+        rating: '4.5',
+        features: productInfo.features
+      }));
+
+    helpers.getProductStyles(this.props.id)
+      .then(productStyles => helpers.findDefaultStyle(productStyles.results)
```

Comment:
You can cut out one of these .then() calls by making findDefaultStyle() synchronous (see my comment below)